java/openjdk17-jetbrains/Portfile

```diff
@@ -2,8 +2,11 @@
 
 PortSystem       1.0
 PortGroup        github 1.0
+PortGroup        deprecated 1.0
 
-github.setup     JetBrains JetBrainsRuntime 17.0.12-b1087.25 jbr-release-
+deprecated.eol_version yes
+
+github.setup     JetBrains JetBrainsRuntime 17.0.14-b1367.22 jbr-release-
 # Change github.tarball_from to 'releases' or 'archive' next update
 github.tarball_from tarball
```

Comment:
please act upon this comment

Reply:
This option isn't used, so I've removed it: https://github.com/macports/macports-ports/pull/29454/commits/190e593920eb9ac5bec8f6a252fde8c95cbd17f6

Reply:
the option is certainly used; if you remove it then it will use the default set by the `github` PortGroup, which is `releases`. If that's correct then I guess it's okay to leave it out. Removing it whould typically have required you though to update the `checksums`, so I am not sure why this appears to work here without it...